usagov/usagov-2021
# USAGOV-1964-usagov-wizard-readme: Update wizard documentation

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1964

## Description


## Type of Changes

  - Add screenshot showing what it should look like
- [ X ] Documentation

## Testing Instructions


1. Check the added doc comments and revised readme.
2. The comments provide useful context for the code.
3. Uninstall the usagov_wizard module.
4. Uninstall the the hierarchy_manager module.
5. Install the usagov_wizard module.
6. You are prompted to install the hierarchy manager module.
7. Respond yes.
8. Both modules install as expected.
9. Navigate to /admin/structure/taxonomy/manage/wizard/overview.
10. The page loads as expected.

### Change Requirements



## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
3. Find the commit of this pull request.
11. Build and deploy the changes.
12. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
